di_website/common/templatetags/string_utils.py

```diff
@@ -50,3 +51,15 @@ def richtext_strip_wrapper(value):
         html = expand_db_html(value)
 
     return mark_safe(html)
+@register.filter(expects_localtime=True, is_safe=False)
+def event_date(start, end=None):
+    
+    """Check that years are not the same and format event date field."""
+
+    if start in (None, '') or end in (None, ''):
+        return ''
+
+    if formats.date_format(start,'Y') == formats.date_format(end,'Y'):
```

Comment:
Install `pylint` to catch basic formatting issues like code spacing.

Reply:
Installed this; seems vscode runs this on save; just need to check problems tab for issues.